src/js/node/events.ts

```diff
@@ -61,8 +61,12 @@ function EventEmitter(opts) {
   }
 
   this._maxListeners ??= undefined;
-  if ((this[kCapture] = opts?.captureRejections ? Boolean(opts?.captureRejections) : EventEmitterPrototype[kCapture])) {
+  if (opts?.captureRejections) {
+    validateBoolean(opts.captureRejections, "options.captureRejections");
```

Comment:
```suggestion
    // TODO: make validator functions return the validated value instead of validating and then coercing an extra time
    validateBoolean(opts.captureRejections, "options.captureRejections");
```